config/locales/en.yml

```diff
@@ -333,6 +333,7 @@ en:
         meta_description: Meta Description
         front_page: Front Page
         active: Active
+        public: Public Category Content
```

Comment:
I think the radio button providing _public_ and _internal_ values, just add more "clean-way" to use this feature. I will provide this change on next commit! Thanks :smiley: